### PR TITLE
Sorted out sponsors

### PR DESCRIPTION
**Pull request by asgrim**
Yep, sort them out

### CODE REVIEW

File: module/Phph/Site/view/phph/sponsors/index.phtml [hunk 1]
```diff
@@ -17,27 +17,13 @@ $this->headTitle($title);
         </li>            
 
         <li class="sponsor">
-            <a href="http://www.protected.co.uk/"><img src="/images/protected.png" alt="Protected.co.uk" class="sponsor-logo" /></a>
-            <h2>Protected.co.uk</h2>
-            <p>Protected.co.uk is a life insurance comparison website which helps customers find the right policy for their needs.</p>
+            <a href="https://roave.com/"><img src="/images/roave.png" alt="Roave" class="sponsor-logo" /></a>
+            <h2>Roave</h2>
+            <p>Roave is a full-service web development firm, offering services such as consulting, training, software development, and more. Roave employs some of the most recognized and accomplished experts in the industry to ensure that organizations have access to the talent they need, when they need it.</p>
             <p>Visit: <a href="http://www.protected.co.uk/">www.protected.co.uk</a></p>
```
Comment (ChiefWigs1982) [inline module/Phph/Site/view/phph/sponsors/index.phtml]: Free advertising? :p

Comment (asgrim) [inline module/Phph/Site/view/phph/sponsors/index.phtml]: Nope, Roave are going to be pizza/beer sponsors :smile:

Comment (lboynton) [inline module/Phph/Site/view/phph/sponsors/index.phtml]: :grinning:

Comment (ChiefWigs1982) [inline module/Phph/Site/view/phph/sponsors/index.phtml]: I was commenting on the link to Protected still being in there. ;)

Comment (asgrim) [inline module/Phph/Site/view/phph/sponsors/index.phtml]: oh nice, didn't spot that - thank you!

**Pull request merged**